fvtest/compilertest/control/TestJit.cpp

```diff
@@ -161,7 +167,7 @@ initializeTestJit(TR_RuntimeHelper *helperIDs, void **helperAddresses, int32_t n
 
    initializeAllHelpers(jitConfig, helperIDs, helperAddresses, numHelpers);
 
-   if (commonJitInit(fe, options) < 0)
+   if (commonJitInit(fe, const_cast<char *>(optionsWithVerifier.c_str())) < 0)
```

Comment:
Scanning through the code quickly, I think this ugly but safe. I agree that `const_cast` is better than c-style cast.